install.php

```diff
@@ -9,7 +9,8 @@
 if(!defined('DOKU_CONF')) define('DOKU_CONF',DOKU_INC.'conf/');
 if(!defined('DOKU_LOCAL')) define('DOKU_LOCAL',DOKU_INC.'conf/');
 
-require_once(DOKU_INC.'inc/PassHash.class.php');
+// load and initialize the core system
+require_once(DOKU_INC.'inc/init.php');
```

Comment:
I do not remember the exact reasons. I think we didn't want to have the full DokuWiki dependency tree in the installer because parts of that dependencies might depend on the settings done in the installer. However I don't see what those would be, so maybe it's time to part with that philosophy.